core/nilai.go

```diff
@@ -0,0 +1,28 @@
+package core
+
+import (
+	"context"
+
+	"github.com/tokopedia/code-review-workshop/errors"
+	"github.com/tokopedia/code-review-workshop/tracer"
+)
+
+type NilaiSiswa struct {
+	NIM     int64  `json:"nim"      db:"nim"`
+	MapelID int64  `json:"mapel_id" db:"mapel_id"`
```

Comment:
```suggestion
	MapelID int  `json:"mapel_id" db:"mapel_id"`
```
sepertinya data type int64 terlalu berlebihan untuk mata pelajaran